src/lib/components/primitives/form/form.svelte

```diff
@@ -55,6 +55,11 @@
           let messageKey = input.getAttribute('data-message');
           input.setAttribute('aria-describedby', messageKey || '');
 
+          if (messageKey) {
+            const messageEl = document.getElementById(messageKey)
+            messageEl?.setAttribute('data-error', 'true');
+          }
```

Comment:
I could set a class on the message element, rather than a data attribute, but I'm on the fence.